### ISSUE TITLE
moscaler not scaling other instances when an instance fails

### ISSUE THREAD

**Issue by djcp** [harvard-dce/mo-scaler#3]
So workers10 failed to provision because of what looks like an ephemeral issue connecting to the ubuntu repositories.  This left moscaler in a state where it gave up and didn't scale up at all, with messages like:

```
Jan 11 08:30:02 monitoring-master1 info: Cluster does not have 8 to start!
```

I think this is as expected, but probably not the behavior we want.

**Comment by lbjay** [comment]
Yep, pretty sure I raised this at some point during development but we punted it for later. Should be fairly easy to adjust the behavior if there's consensus.
